www.foia.gov/_sass/modules/_react_jsonschema_form.scss

```diff
@@ -0,0 +1,6 @@
+// Customization of the form elements generated from react-jsonschema-form
+
+.field-description {
```

Comment:
Will we need this on other pages as well, or is this the only one we're creating with `react-jsonschema`? (I *think* it's the only one, but if it's not, we should move this to a more general spot.)

Reply:
Ah, good point. Yeah, this is a react-jsonschema-form specific class name, but if we have forms elsewhere that have help text, we probably want them styled the same. If that comes up, I would move this to a forms.scss as a template class and then have `.field-description { @extend %form-field-description; }` or something. I think you're right that this is the only one so I'm going to leave this as is for now.